npm/cmd/start.go

```diff
@@ -85,6 +85,9 @@ func newStartNPMCmd() *cobra.Command {
 
 	startNPMCmd.Flags().String(flagKubeConfigPath, flagDefaults[flagKubeConfigPath], "path to kubeconfig")
 
+	startNPMCmd.AddCommand(newStartNPMControlplaneCmd())
```

Comment:
can we add comments here and/or for newStartNPMDaemonCmd()? Saying that this daemon refers to just the fan-out mode

Reply:
done